smqtk_iqr/web/__init__.py

```diff
@@ -7,14 +7,12 @@
 
 import flask
 
-from smqtk.utils import SmqtkObject
-from smqtk.utils.configuration import Configurable
-from smqtk.utils.dict import merge_dict
-from smqtk.utils.plugin import Pluggable
+from smqtk_core import Configurable, Pluggable
+from smqtk_core.dict import merge_dict
 
 
 # noinspection PyAbstractClass
-class SmqtkWebApp (SmqtkObject, flask.Flask, Configurable, Pluggable):
+class SmqtkWebApp (flask.Flask, Configurable, Pluggable):
```

Comment:
Could make use of `Plugfigurable` instead of the two classes. Not specifically important, just syntactic sugar.